src/Modal/Modal.js

```diff
@@ -196,9 +197,8 @@ Modal.Body = createComponent({
 
 Modal.Footer = createComponent({
   name: 'ModalFooter',
-  style: ({ theme }) => css`
-    padding: 1rem 1.25rem;
-    background: ${theme.colors.greyLightest};
+  style: css`
+    padding: 8px;
```

Comment:
Buttons feel a little tight - should be same padding as header?
<img width="389" alt="Screen Shot 2019-09-17 at 9 46 16 AM" src="https://user-images.githubusercontent.com/14184138/65062056-10801d80-d930-11e9-925e-cf1d5767ead1.png">